google-cloud-testing/storage-benchwrapper/src/main/java/com/google/cloud/benchwrapper/StorageBenchWrapperImpl.java

```diff
@@ -32,10 +32,16 @@
   private Storage client;
 
   public StorageBenchWrapperImpl(String storageEmulatorHost) {
-    client = StorageOptions.newBuilder()
+    if (storageEmulatorHost.isEmpty()){
```

Comment:
Normally this would be a null check (or both null and empty) versus an empty string